eng/versioning/version_client.txt

```diff
@@ -67,7 +67,7 @@ com.azure:azure-aot-graalvm-support-netty;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-aot-graalvm-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-aot-graalvm-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-code-customization-parent;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-callautomation;1.2.3;1.3.0-beta.2
+com.azure:azure-communication-callautomation;1.3.0-beta.1,1.3.0-beta.2
```

Comment:
The released version should always be a GA version (when available). So, in this case, 1.3.0-beta.1 should be updated to 1.2.4.

Reply:
Fixed!